manhunt-base/src/main/kotlin/io/github/ytg1234/manhunt/util/ManhuntServer.kt

```diff
@@ -0,0 +1,24 @@
+package io.github.ytg1234.manhunt.util
+
+import net.minecraft.entity.player.PlayerEntity
+import java.util.*
```

Comment:
Please don't use star imports

Reply:
Sorry, but stupid IntelliJ did it.

Reply:
I remember having an editorconfig value preventing that 🤔